scripts/check_dependencies.sh

```diff
@@ -40,16 +40,17 @@ if ! which awk >&/dev/null; then
 fi
 
 if which python >&/dev/null ; then
-  version=`python 2>&1 --version | awk '{print $2}' `
-  if [[ $version != "2."* ]] ; then
-    if which python2.7 >&/dev/null  || which python2 >&/dev/null ; then
-      echo "$0: python 2.x is not the default python. You should either make it"
-      echo "$0: default or create an bash alias for kaldi scripts to run correctly"
+  version=`/usr/bin/env python 2>&1 --version | awk '{print $2}' `
+  if [[ $version != "2.7"* && $version != "3."* ]] ; then
+    if which python2.7 >&/dev/null ; then
+      echo "$0: python 2.7 is not the default python (lower version python does not "
+      echo "$0: have packages that are required by pocolm). You should make it default"
```

Comment:
you should set error status on this branch.
you may have to move 'status=0' higher in the script.